src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -185,6 +185,55 @@ export const exchangeStitchingEnvironment = ({
     },
   }
 
+  const paymentMethodResolver = {
+    fragment: gql`
+      fragment CommerceOrderPaymentMethod on CommerceOrder {
+        creditCardId
+        bankAccountId
+        paymentMethod
+      }
+    `,
+
+    resolve: async (parent, _args, context, info) => {
+      const { creditCardId, bankAccountId, paymentMethod } = parent
+
+      if (paymentMethod === "credit card") {
```

Comment:
And finally in this file we add the stitched and enriched payment method to the schema.